packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -48,6 +48,7 @@
                     <form-field
                         :value="customer.mobileNumber"
                         name="mobile-number"
+                        :input-type="tel"
```

Comment:
```suggestion
                        input-type="tel"
```
Should just be a string rather than a v-bind